app/src/main/java/com/asfoundation/wallet/ui/iab/MergedAppcoinsFragment.kt

```diff
@@ -63,7 +66,7 @@ class MergedAppcoinsFragment : DaggerFragment(), MergedAppcoinsView {
                     productName: String?,
                     appcAmount: BigDecimal, appcEnabled: Boolean,
                     creditsEnabled: Boolean, isBds: Boolean,
-                    isDonation: Boolean): Fragment {
+                    isDonation: Boolean, skuId: String, transactionType: String): Fragment {
       val fragment = MergedAppcoinsFragment()
       val bundle = Bundle()
```

Comment:
This is not wrong but in kotlin you can use do the same by saving "bundle." in every line:

val bundle = Bundle().apply {
        putSerializable(FIAT_AMOUNT_KEY, fiatAmount)
        putString(FIAT_CURRENCY_KEY, currency)
        putString(BONUS_KEY, bonus)
        putString(APP_NAME_KEY, appName)
        putString(PRODUCT_NAME_KEY, productName)
        putSerializable(APPC_AMOUNT_KEY, appcAmount)
        putBoolean(APPC_ENABLED_KEY, appcEnabled)
        putBoolean(CREDITS_ENABLED_KEY, creditsEnabled)
        putBoolean(IS_BDS_KEY, isBds)
        putBoolean(IS_DONATION_KEY, isDonation)
        putString(SKU_ID, skuId)
        putString(TRANSACTION_TYPE, transactionType)
      }